packages/web3-providers-ws/package.json

```diff
@@ -14,10 +14,12 @@
   "types": "types/index.d.ts",
   "main": "lib/index.js",
   "dependencies": {
+    "@nomiclabs/websocket": "^1.0.31",
     "eventemitter3": "4.0.4",
+    "i": "^0.3.6",
```

Comment:
I think you run `npm i npm i @nomiclabs/websocket` 😅

Reply:
Omg lol...